test/test_bson.py

```diff
@@ -980,7 +980,8 @@ def test_codec_options_repr(self):
             "uuid_representation=UuidRepresentation.UNSPECIFIED, "
             "unicode_decode_error_handler='strict', "
             "tzinfo=None, type_registry=TypeRegistry(type_codecs=[], "
-            "fallback_encoder=None), datetime_conversion=1)"
+            "fallback_encoder=None), "
+            "datetime_conversion=<DatetimeConversionOpts.DATETIME: 1>)"
```

Comment:
CodecOptions's repr tries to be able to be in a format that can be eval'd so in this case CodecOptions's repr should use `%s` for datetime_conversion. We would expect this line to be:
```

            "datetime_conversion=DatetimeConversionOpts.DATETIME)"
```